### PR TITLE
Rename object to avoid naming conflicts

### PR DESCRIPTION
**Pull request by dermatobia**
Rename object to avoid naming conflicts with `disableProgress` function.

### CODE REVIEW

**Comment by davidjamesherzog** [comment]
You need to fix up the tests as well to reflect your changes.  Run `npm test` and you will see the tests fail.

**Comment by dermatobia** [comment]
Good thinking. Forgot about that. 
I have updated the tests and readme file.

**Comment by davidjamesherzog** [comment]
@jinpark - wondering if we should tick the version here or do we even care since we are taking it over?

**Comment by jinpark** [comment]
im ok either way. makes sense to tick the version to kinda show that we took it over?

**Comment by dermatobia** [comment]
How do we do that? 
Is it something that has to be done on the repo or do you mean update the versions of these two files to something like this: ```"version": "0.2.0"``` ?

https://github.com/DramaFever/videojs-disable-progress/blob/master/package.json#L3
https://github.com/DramaFever/videojs-disable-progress/blob/master/bower.json#L3

File: test/disableProgress.test.js [hunk 1]
```diff
@@ -78,7 +78,7 @@
           autoDisable: false
         });
 
-        strictEqual(this.player.disableProgress.getState(), false, 'returns false');
+        strictEqual(this.player.controlProgress.getState(), false, 'returns false');
     });
 
-}(window.videojs));
\ No newline at end of file
+}(window.videojs));
```
Comment (jinpark) [inline test/disableProgress.test.js]: 👍

**Comment by jinpark** [comment]
updating the version in those files should be good enough afaik

**Comment by davidjamesherzog** [comment]
LGTM

**Pull request merged**